projects/ngrx.io/content/guide/component-store/effect.md

```diff
@@ -91,3 +91,24 @@ An easy way to handle the response in ComponentStore effects in a safe way, with
     );
   });
 </code-example>
+
+## Calling an `effect` without parameters
+
+A common use case is to call the `effect` method without any parameters. 
+To make this possible:
+- The generic type of the `effect` method needs to be set to `void`.
+- The name of the source stream doesn't matter: `trigger$`, `source$` or `$` are good names.
+
+<code-example header="movies.store.ts">
+  readonly getAllMovies = this.effect<void>(
+    // Can be named whatever, for example source$. The name doesn't matter.
+    trigger$ => trigger$.pipe(
+      exhaustMap(() => this.moviesService.fetchAllMovies().pipe(
+        tapResponse(
+          movies => this.addAllMovies(movies),
+          (error: unknown) => this.logError(error),
+        )
+      )
+    )
+  );
```

Comment:
We can remove additional nesting and `unknown` type for `error` because it will be `unknown` by default. Also, we need to use `&lt;` and `&gt;` for angle brackets to display them properly.

The current version:

![cs-effect-wo-params](https://user-images.githubusercontent.com/17877290/210256859-5a920db9-bc9e-4b9d-8cb1-6c4fb5c35937.png)

```suggestion
readonly getAllMovies = this.effect&lt;void&gt;(
  (trigger$) => trigger$.pipe(
    exhaustMap(() => this.moviesService.fetchAllMovies().pipe(
      tapResponse(
        (movies) => this.addAllMovies(movies),
        (error) => this.logError(error)
      )
    )
  )
);
```